src/runtime_src/core/pcie/linux/pcidev.h

```diff
@@ -78,22 +69,18 @@ class pci_device
 
   uint16_t domain =           INVALID_ID;
   uint16_t bus =              INVALID_ID;
-  uint16_t dev =              INVALID_ID;
+  uint16_t dev_no =           INVALID_ID;
   uint16_t func =             INVALID_ID;
-  uint16_t vendor_id =        INVALID_ID;
-  uint16_t device_id =        INVALID_ID;
   uint32_t instance =         INVALID_ID;
   std::string sysfs_name =    ""; // dir name under /sys/bus/pci/devices
```

Comment:
By default strings are initialized to empty strings.  So this extra assignment isn't needed.
```suggestion
  std::string sysfs_name;; // dir name under /sys/bus/pci/devices
```

Reply:
removed.